code/_globalvars/global_lists.dm

```diff
@@ -63,6 +63,19 @@ GLOBAL_LIST_INIT(resin_build_order_ovipositor, list(
 	/datum/resin_construction/resin_obj/resin_spike
 ))
 
+//Resin constructions that will be SMUSHED by closing doors
```

Comment:
Don't need to make this a global var if you're gonna use it in 1 proc. Make it a static or something

Reply:
ok